### PR TITLE
[Improve] Fix the typo of "Metrics", and fix the npe when jobStatus is null.

### PR DESCRIPTION
**Pull request by FlechazoW**
## Purpose of this pull request

Fix the typo of "Metrics", and fix the npe when jobStatus is null.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/JobMetricsServiceImpl.java [hunk 1]
```diff
@@ -183,17 +183,17 @@ private Map<Long, JobSummaryMetricsRes> getMatricsListIfTaskTypeIsBatch(
...
                                     Long.toString(
                                             jobInstanceIdAndJobEngineIdMap.get(
                                                     jobInstance.getId())));
-                    if (jobMetriceFromDb != null) {
-                        jobSummaryMetricsResMap.put(jobInstance.getId(), jobMetriceFromDb);
+                    if (jobMetricsFromDb != null) {
+                        jobSummaryMetricsResMap.put(jobInstance.getId(), jobMetricsFromDb);
                     }
-                    if (jobInstance.getJobStatus().equals("RUNNING")) {
+                    if ("RUNNING".equals(jobInstance.getJobStatus())) {
```
Comment (XiaoJiang521) [inline seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/JobMetricsServiceImpl.java:196]: Now that you've fixed this problem, how about StringUtils.euqles here

Comment (XiaoJiang521) [reply seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/JobMetricsServiceImpl.java:196]: Of course, it's perfectly fine to leave it here

Comment (FlechazoW) [reply seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/JobMetricsServiceImpl.java:196]: I think the code is clean and easy-read, using StringUtils adds complexity.

Comment (XiaoJiang521) [reply seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/JobMetricsServiceImpl.java:196]: Yes, you don't have to change it here

**Comment by XiaoJiang521** [comment]
LGTM

**Pull request closed without merging**